src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -58,11 +58,12 @@ static int        CCSN_Eint_Mode;                  // Mode of obtaining internal
        double     CCSN_CC_Red_DT;                  // reduced time step (in s) when the central density exceeds CCSN_CC_CentralDensFac before bounce
        double     CCSN_MaxRefine_RadFac;           // factor that determines the maximum refinement level based on distance from the box center
        double     CCSN_LB_TimeFac;                 // factor that scales the dt constrained by lightbulb scheme
-       bool       CCSN_CC_Rot;                     // add formulated shell-type initial angular velocity
+       bool       CCSN_Is_Rotation = false;        // boolean that indicates whether we add rotational profile
```

Comment:
Why initialize `CCSN_Is_Rotation = false` here?

Reply:
This variable is merged to `CCSN_CC_Rot`